src/mame/taito/taito_f3.h

```diff
@@ -310,8 +317,7 @@ class taito_f3_state : public driver_device
 
 	void tile_decode();
 
-	inline void f3_drawgfx(bitmap_rgb32 &dest_bmp, const rectangle &clip, gfx_element *gfx, int code, u8 color, bool flipx, bool flipy, int sx, int sy, u8 pri_dst);
-	inline void f3_drawgfxzoom(bitmap_rgb32 &dest_bmp, const rectangle &clip, gfx_element *gfx, int code, u8 color, bool flipx, bool flipy, int sx, int sy, u16 scalex, u16 scaley, u8 pri_dst);
+	inline void f3_drawgfx(bitmap_rgb32 &dest_bmp, const rectangle &clip, gfx_element *gfx, const int code, const u8 color, const bool flipx, const bool flipy, int sx, int sy, u16 scalex, u16 scaley, u8 pri_dst);
```

Comment:
What’s the performance impact of getting rid of the non-zooming version of this function?  The F3 graphics code can be pretty heavy, so having an optimised version for the 1:1 case could still be beneficial.

Reply:
I cloned the current master branch of mame and ran
```
taitoem -bench 12 arabianm
taitoem -bench 120 arabianm
taitoem -bench 120 elvactr
taitoem -bench 120 gekiridn
taitoem -bench 120 landmakr
taitoem -bench 120 lightbr
taitoem -bench 120 puchicar
taitoem -bench 120 quizhuhu
taitoem -bench 120 ridingf
```
twice, taking an average of all benches (minus the -bench 12 arabianm, that was the "warmup"). Then i removed `drawgfx()` and ran it again.

**average speed**
mame-master: 295.71%
fixed scale removed: 298.17%

My best guess for this result is that `drawgfxzoom` has newer style code.